set-target.sh

```diff
@@ -15,7 +15,7 @@ if [[ "$VERSION" == /* ]]; then
 else
     $GOCC mod edit -dropreplace=github.com/ipfs/go-ipfs
     # Resolve the exact version/package name
-    MODFILE="$(go list -f '{{.GoMod}}' -m "$PKG@$VERSION")"
+    MODFILE="$(go list -mod=mod -f '{{.GoMod}}' -m "$PKG@$VERSION")"
```

Comment:
I thought it was required since I always got the following error when removing that:
```
go: github.com/ipfs/go-ipfs@v0.7.0: missing go.sum entry; to add it:
        go mod download github.com/ipfs/go-ipfs
make: *** [Makefile:26: go.mod] Error 1
```
However, I found that clearing my mod cache (`go clear -modcache`) also made the error disappear.  So, apparently it is not necessary.

Reply:
The test is still not passing due to:
```
Error: error loading plugins: loading plugin /home/circleci/.ipfs/plugins/go-ds-s3.so: plugin.Open("/home/circleci/.ipfs/plugins/go-ds-s3"): plugin was built with a different version of package golang.org/x/net/html/atom
```
I have tried building with the version of the aws lib that uses the same version of `golang.org/x/net` that was used to build go-ipfs v0.8.0, but have not had any success.  Maybe I need to explicitly require a particular version of `golang.org/x/net`, although the plugin does not require it itself.